contracts/contracts/Suffragium.sol

```diff
@@ -103,11 +83,28 @@ contract Suffragium is ISuffragium, IdentityManager, GatewayCaller, Ownable {
         return _castedVotes[voteId][voterId];
     }
 
+    /// @notice Returns the number of votes cast for a specific vote
+    /// @param voteId ID of the vote to get the count for
+    /// @return The number of votes cast
+    function getVoteCount(uint256 voteId) external view returns (uint256) {
```

Comment:
i would avoid to create this function as it's possible to simply call `vote(voteId)`

Reply:
Sure can delete them, they were just helper functions for testing